egov-persister/inportal-survey-persister.yml

```
@@ -0,0 +1,90 @@
+            - jsonPath: $.feedBack.uuid
+            - jsonPath: $.feedBack.rating
+            - jsonPath: $.feedBack.category
+            - jsonPath: $.feedBack.tenantId
+            - jsonPath: $.feedBack.feedback
+            - jsonPath: $.feedBack.auditDetails.createdBy
+            - jsonPath: $.feedBack.auditDetails.lastModifiedBy
+            - jsonPath: $.feedBack.auditDetails.createdTime
+            - jsonPath: $.feedBack.auditDetails.lastModifiedTime
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 0)

(empty-lines)